src/All-Movies/All-Movies.scss

```diff
@@ -0,0 +1,21 @@
+@import '../styles.scss';
+
+.movies-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-gap: 10px;
+    background-color: $dark-grey;
+    padding: 20px 0 20px 0;
+    font-size: 150%;
+    height: auto;
+    width: 100vw;
+}
+
+.movies-container>img {
+    height: auto;
+    width: 80%;
+    &:hover {
```

Comment:
what do you think about dark-grey as the background? I'm not married to it maybe the off-black would look better? What do you think?

Reply:
The image seems to have a default padding that is making the drop shadow not be directly
 behind the image. Could you add a property of `padding: 0` and `margin: 20px` so it still has the same spacing between each movie, but so the drop shadow is only behind the image? I made an example in the dev tools of what the change would look like:
<br>
![Rancid Tomatillos](https://user-images.githubusercontent.com/95593550/187079599-6b9bd57a-6959-4d30-96b2-38b4ee298e8e.gif)

Reply:
Will do! That looks a lot better :o